packages/design-system/src/components/Table/TableCell.jsx

```diff
@@ -13,18 +14,25 @@ export const TableCell = ({
   stackedTitle,
   stackedClassName,
   _isTableHeadChild,
-  _stackable,
   ...tableCellProps
 }) => {
-  if (process.env.NODE_ENV !== 'production' && _stackable) {
+  const stackableTable = useContext(TableContext);
+
+  let Component;
+  if (component) {
+    Component = component;
+  } else {
+    Component = _isTableHeadChild ? 'th' : 'td';
+  }
+  if (process.env.NODE_ENV !== 'production' && stackableTable) {
```

Comment:
@nichia, I don't have time to test locally with MPF at the moment, but I wonder if you can add a boolean prop (or 2, but I think 1 might be fine) that allows the users to just disable these warnings altogether. The reason I want to suppress them is that they can greatly slow things down during development, and I think we have cases where we may "violate the rules" because we need to implement things differently for a stacked table, particularly when headers contain JSX and not just simple strings of text.

So, I think these changes look fine, but would like to have the option to suppress warnings via (a) prop(s), if possible. Thanks

Reply:
Yes, we can add a disable warning prop which would helpful with the cases that you mentioned, especially when working with large datasets. One thing to note, since the id prop associates header and data cells for screen readers in stacked format, it's recommended that accessibility testing with screen readers is done to make sure we're not loosing out on that.

Reply:
Thanks. Hopefully after we get through our current work on Plan Details redesign, we can work on refining our implementation to ensure it's accessible, and I can test with the updated `Table`. If there are still issues to be worked out, we can address at that time. The point of the prop(s) would be to allow the user to acknowledge they know what they're doing and that they will do their best to ensure good accessibility. For that matter, you can annotate the prop(s) to say as much.